Project/Source/main.c

```diff
@@ -1,5 +1,101 @@
 #include "main.h"
+#include <stdlib.h>
 
-int main (void){
-	while(1);
+void Delay(void);
+
+int main(void)
+
+{
+	int random1, random2, i;
+	int postojiMetak = 0;
+	PlayerPosition player1, player2;
+	Queue queue;
+	BulletPosition temp;
+	
+	queue.end = -1; 
+	queue.top = -1;
+	
+	StartScreen();
+	
+	Delay();
+	
+	ClearScreen();
+	
+	player1 = TankInit(0);
+	player2 = TankInit(1);
+	
+	Delay();
+	
+  while (1){
```

Comment:
ovo je test da se vidi kako funkcionira kretanje tenka?

nemam stm s ekranom pa ne mogu vidit sta se desava

Reply:
Da u mainu je petlja sa randomiziranim kretanjem/pucanjem/rotiranjem oba tenka da se vidi kako to otprilike radi.